src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball extends ColorSupplier {
```

Comment:
Ball shouldn't extend ColorSupplier.
It should be simple model with constructor/getters/setters required